packages/salesforcedx-vscode-core/src/commands/functions/forceFunctionInvokeCodeLensProvider.ts

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import * as path from 'path';
+import {
+  CancellationToken,
+  CodeLens,
+  CodeLensProvider,
+  Command,
+  EventEmitter,
+  ExtensionContext,
+  languages,
+  Position,
+  Range,
+  TextDocument
+} from 'vscode';
+import { nls } from '../../messages';
+import { FUNCTION_PAYLOAD_DOCUMENT_SELECTOR } from './types/constants';
+
+/**
+ * Code Lens Provider providing "Invoke"
+ */
+class ForceFunctionInvokeCodeLensProvider implements CodeLensProvider {
+  private onDidChangeCodeLensesEventEmitter = new EventEmitter<void>();
+  public onDidChangeCodeLenses = this.onDidChangeCodeLensesEventEmitter.event;
+
+  /**
+   * Refresh code lenses
+   */
+  public refresh(): void {
+    this.onDidChangeCodeLensesEventEmitter.fire();
+  }
+
+  /**
+   * Invoked by VS Code to provide code lenses
+   * @param document text document
+   * @param token cancellation token
+   */
+  public async provideCodeLenses(
+    document: TextDocument,
+    token: CancellationToken
+  ): Promise<CodeLens[]> {
+    return provideFunctionInvokeCodeLens(document, token);
+  }
+}
+
+export const functionInvokeCodeLensProvider = new ForceFunctionInvokeCodeLensProvider();
+
+/**
+ * Register Code Lens Provider with the extension context
+ * @param context Extension context
+ */
+export function registerFunctionInvokeCodeLensProvider(context: ExtensionContext) {
```

Comment:
This part of the code fails to register the code lens in the correct file if more than one *.json file is present. If a user builds the function it will generate a package-lock.json file and the code lens will get assigned to that one instead of the function's `package.json`

Reply:
![send-request](https://user-images.githubusercontent.com/1041842/93117636-fe84a800-f673-11ea-9a34-0400aa575960.png)

Reply:
I've now included 'package.json', 'package-lock.json', 'tslint.json', 'lerna.json', 'tsconfig.json' as the list of non-payload files. Let me know I should include any more.